apps/playground/app/test-data-list/page.tsx

```diff
@@ -0,0 +1,228 @@
+                  <DataListData>Obi wan</DataListData>
+                </DataListItem>
+                <DataListItem>
+                  <DataListLabel>Padewan</DataListLabel>
+                  <DataListData>Anakin Skywalker</DataListData>
+                </DataListItem>
+              </DataListRoot>
+            </Flex>
+            <Flex gap="2" mt="4" direction="column">
+              <Heading mb="5" size="3">
+                Vertical Layout
+              </Heading>
+              <DataListRoot direction="column" mt="2">
+                <DataListItem>
+                  <DataListLabel>Jedi Master</DataListLabel>
+                  <DataListData>Obi wan</DataListData>
+                </DataListItem>
+                <DataListItem>
+                  <DataListLabel>Padewan</DataListLabel>
+                  <DataListData>Anakin Skywalker</DataListData>
+                </DataListItem>
+              </DataListRoot>
+            </Flex>
+
+            <Box mb="6" mt="6">
+              <Heading mb="5" size="3">
+                With varied content
+              </Heading>
+              <DataListRoot direction={{ initial: 'column', sm: 'row' }}>
+                <DataListItem>
+                  <DataListLabel>Status</DataListLabel>
+                  <DataListData>
+                    <Badge color="green" size="1" style={{ marginLeft: -2 }}>
+                      Active
+                    </Badge>
+                  </DataListData>
+                </DataListItem>
+
+                <DataListItem align="center">
+                  <DataListLabel>Name</DataListLabel>
+                  <DataListData>
+                    <Button size="2">Add</Button>
+                  </DataListData>
+                </DataListItem>
+
+                <DataListItem>
+                  <DataListLabel>Email</DataListLabel>
+                  <DataListData>vlad@workos.com</DataListData>
+                </DataListItem>
+
+                <DataListItem>
+                  <DataListLabel>Organization</DataListLabel>
+                  <DataListData>
+                    <Link href="https://workos.com">WorkOS</Link>
+                  </DataListData>
+                </DataListItem>
+
+                <DataListItem>
+                  <DataListLabel>Long value</DataListLabel>
+                  <DataListData>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla ac nisl et libero
+                    ultricies viverra quis vitae quam. Proin a feugiat metus.
+                  </DataListData>
+                </DataListItem>
+
+                <DataListItem align="center">
+                  <DataListLabel>Authentication methods</DataListLabel>
+                  <DataListData>
+                    <Flex gap="2" align="center" height="100%">
+                      <StarIcon />
+                      <MagicWandIcon />
+                      <StarFilledIcon />
+                    </Flex>
```

Comment:
@vladmoroz https://github.com/radix-ui/themes/pull/231#discussion_r1453793293, we need to set the align on the `Flex` parent for this case which I think isn't odd given how flex works. Lmk your thoughts.